Vostok.Applications.AspNetCore/Builders/VostokKestrelBuilder.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using Microsoft.AspNetCore.Server.Kestrel.Core;
+using Microsoft.AspNetCore.Server.Kestrel.Transport.Sockets;
+using Vostok.Applications.AspNetCore.Configuration;
+using Vostok.Commons.Helpers;
+
+namespace Vostok.Applications.AspNetCore.Builders
+{
+    internal class VostokKestrelBuilder
+    {
+        private readonly Customization<KestrelSettings> kestrelCustomization = new Customization<KestrelSettings>();
+
+        public void Customize(Action<KestrelSettings> customization)
+            => kestrelCustomization.AddCustomization(customization);
+
+        public void ConfigureKestrel(KestrelServerOptions options)
+        {
+            var settings = kestrelCustomization.Customize(new KestrelSettings());
+
+            options.AddServerHeader = false;
```

Comment:
А как менять эти константы?

Reply:
You can do this using `services.Configure<KestrelServerOptions>(...)` in `ConfigureServices` section of Startup. 

For example this will work (checked myself for AddServerHeader) https://docs.microsoft.com/en-us/aspnet/core/fundamentals/servers/kestrel?view=aspnetcore-3.1#kestrel-options